website/vercel.json

```
@@ -3632,11 +3632,6 @@
       "destination": "/reference/dbt-jinja-functions/adapter",
       "permanent": true
     },
```

We might want to forward the page in case someone has it bookmarked.
```suggestion
    },
    {
      "source": "/reference/dbt-jinja-functions/as_text",
      "destination": "/reference/dbt-jinja-functions",
      "permanent": true
    },
```